tools/runs-to-csv.sh

```diff
@@ -61,7 +61,8 @@ find . -name benchmark-tekton.json -print0 | while IFS= read -r -d '' filename;
         .measurements.apiserver.cpu.mean,
         .measurements.apiserver.memory.mean,
         .measurements."kube-apiserver".cpu.mean,
-        .measurements."kube-apiserver".memory.mean
+        .measurements."kube-apiserver".memory.mean,
+        .measurements.tekton_tekton_chains_controller_workqueue_depth.mean
```

Comment:
Could you put it next to `.measurements.tekton_tekton_pipelines_controller_workqueue_depth.mean`?